hw4/js/to-do.js

```diff
@@ -0,0 +1,52 @@
+//add the input to the to-do list when clicked
+$(document).ready(
+    $("#new-item").on('click', function() {
+        var user_input = $("#item-input").val();
+        $("#to-do-list").prepend("<div> <button class='to-do-item'>"+ "Mark" + "</button> " + user_input + "</div>");
+    })
+);
+
+//when the button of to-do list item take actions 1)move it to complete list, 2) changestyle, 3) change button content
+$("#to-do-list").on('click', 'div', function() {
```

Comment:
One of the most important principles of writing effective code is DRY (Don't Repeat Yourself). Lines 10-16 and lines 19-25 are very similar. Can you have designed this so that you don't repeat yourself. This book is a great resource on the topic check it out: https://addyosmani.com/resources/essentialjsdesignpatterns/book/#antipatterns. I would read the section on anti-patterns.